.circleci/config.yml

```diff
@@ -105,16 +85,24 @@ jobs:
       TEST_DIR: "integration/test/clusterdeletion"
     <<: *e2eTest
 
-
 workflows:
   version: 2
   build_and_e2eTest:
     jobs:
       # build binary
+      - architect/go-test-legacy:
+          name: test
       - build:
+          requires:
+            - test
+      - architect/go-architect-legacy:
```

Comment:
I'm not sure what architect deploy does for azure-operator, but since there wasn't a `./architect deploy` step before, I think this isn't necessary here.

Reply:
Removed